release_images.yml

```diff
@@ -2,9 +2,25 @@
 release_images:
   1:
     framework: "tensorflow"
-    version: "2.2.0"
-    inference:
+    version: "1.15.3"
+    training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py37", "py36"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu102"
+      cuda_version: "cu100"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: True  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
```

Comment:
just double checking, we are not releasing a sm tag for either py37 or py36?

Reply:
We are releasing SM tags, I have made the required changes